Cargo.toml

```diff
@@ -277,9 +277,25 @@ members = [
 ]
 default-members = ["cmd/tikv-server", "cmd/tikv-ctl"]
 
+[profile.dev.package.grpcio-sys]
+debug = false
+opt-level = 1
+
+[profile.dev.package.librocksdb_sys]
+debug = false
+opt-level = 1
+
+[profile.dev.package.libtitan_sys]
+debug = false
+opt-level = 1
+
+[profile.dev.package.tests]
+debug = 1
+opt-level = 1
+
 [profile.dev]
 opt-level = 0
-debug = true
+debug = 0
```

Comment:
I think it's handy for debugging, let's make it to 1 as least? It's enabled on purpose before https://github.com/tikv/tikv/issues/5049

Reply:
> I think it's handy for debugging

I set the tests to 1 for the exact purpose. So that we can see the exact panic point in tests. Setting global to 1 will make the target directory grow to 20GiB.